src/mongoose/updateOne.ts

```diff
@@ -0,0 +1,36 @@
+import { t } from 'i18next';
```

Comment:
I have to think about this. When I did the i18n work I made a point to always get `t` off of `i18n` on the `req` which is assigned in a middleware. Is it just as well to import it anywhere like this? I'm concerned if people have customized their i18n through the config that it might not pick up the same instance by doing it this way.

Reply:
> I have to think about this. When I did the i18n work I made a point to always get `t` off of `i18n` on the `req` which is assigned in a middleware. Is it just as well to import it anywhere like this? I'm concerned if people have customized their i18n through the config that it might not pick up the same instance by doing it this way.

I think the answer is obvious that it should be done via the req object. 

If it is true that you can customize i18n through the config, and that i18n from the config gets passed through the req through middleware. Then it is best practice to use the req object from the middleware instead of base importing it from an i18next package which might not have the changes from the config!

Reply:
Hm, so req actually isn't passed in the `updateOne` function as an argument. I feel a bit weird making it a mandatory argument. What do you think?

What if I import `payload` and use it from there (shouldn't it be the same as getting it from `req`?), or use some kind of dependency injection for that?

Reply:
> Hm, so req actually isn't passed in the `updateOne` function as an argument. I feel a bit weird making it a mandatory argument. What do you think?
> 
> What if I import `payload` and use it from there (shouldn't it be the same as getting it from `req`?), or use some kind of dependency injection for that?

https://payloadcms.com/docs/local-api/overview#accessing-payload

I believe so, as long as you can guarantee this is being imported after payload.init()